web/src/main/java/org/hillview/targets/TableTarget.java

```diff
@@ -555,6 +556,21 @@ private static TopList getTopList(FreqKList fkList, Schema schema, HillviewCompu
         return tl;
     }
 
+    /**
+     * Post-processing method applied to the result of a heavy hitters sketch before displaying the results. It will
```

Comment:
We have lots of heavy hitters now. Perhaps we should use more specific names for each of them?